starter-code/index.html

```diff
@@ -1,44 +1,122 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="css/main.css" />
     <title>Spotify</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Montserrat:100,400&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    Premium Discover Help Download
+    <div class="navbar">
+      <div>
+        <img
+          class="logo"
+          src="../starter-code/images/spotify-logo.png"
+          alt="logo"
+        />
+      </div>
+      <div class="links">
+        <div class="links">Premium</div>
```

Comment:
best it to make this a `ul` containing `li `tags. Since its a list that you will flex to a row.

Also make sure that they are wrapped around ` a` tags in order for them to become an actual link.